docs/versioned_docs/version-1.0.0/guides/alternative-billing.md

```
@@ -0,0 +1,509 @@
+import GreatFrontEndBanner from '@site/src/uis/GreatFrontEndBanner';
```

markdownlint-cli2 (0.18.1)

First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)